datalad/interface/pull.py

```diff
@@ -39,4 +39,4 @@ class Pull(Interface):
     def __call__(remote='origin'):
 
         repo = get_repo_instance()
-        repo.git_pull()
+        repo.pull()
```

Comment:
so we have not a single test of any kind for pull/push commands here... #492 although not sure if we need at least pull